README.md

```diff
@@ -0,0 +1,36 @@
+# Ô Chap'
+
+Ô Chap' is an e-commerce website for selling all kinds of hats!
+
+## Installation
+
+### Requirements
+
+- Git
+- JDK 11 or later
+- Maven 3.2 or later
+
+### Instructions
+
+Clone the current repository.
+
+Run the project locally with Maven using Spring's built-in development server:
+```bash
+mvn spring-boot:run
+```
+
+You should be able to access the website on [http://localhost:8080/](http://localhost:8080/).
```

Comment:
This endpoint does not exist atm, but I think it's fine since we expect to include it soon.